### DISCUSSION TITLE
Wifi Adaptor not working on Teclast F7 plus

### DISCUSSION THREAD

**Discussion by pmcd** [sebanc/brunch#1641] [Q&A]
Hi All,

First time to post. I followed a tutorial on Youtube on installing Chrome OS. The installation worked except I can't get my wifi adaptors to work. Now I know this is a problem on a lot of install, was also the same problem when I installed Chrome OS Flex. My former windows laptop is 2019 Teclast F7 Plus with Intel Celeron N4100 and Intel Wireless 3165 driver.

I used the latest version of Linux Mint and Brunch r103 stable 20220721 with chromeos_14816.131.0_rammus (used on a ASUS Chromebook C425).

Installation went like a dram no hiccups. 

Can anyone point me in the right direction on how to resolve this? Im a newbie at this learning as I go along... any help is greatly appreciated. the machine running on windows is like trying to peel an orange in your pocket with a boxing glove on, on Chrome (and hard wired to router) its an absolute flyer.

Many thanks folks.

**Comment by fafan999** [comment]
Changing some BIOS settings might help: https://askubuntu.com/a/1387432/1562392
